src/search_handler.cc

```diff
@@ -84,13 +84,13 @@ SearchResults PerformSearch(const SearchOptions& options, RegexTraits traits,
         }
 
         progress_channel->Push(
-            {.counters = {{L"matches", output.positions.size()}}});
+            (ProgressInformation) {.counters = {{L"matches", output.positions.size()}}});
```

Comment:
¡Hola, Sergio! :-) ¡Qué sorpresa!

Drop the parenthesis? Just: progress_channel->Push(ProgressInformation{.counters ....});

Same in the other line.

Reply:
:smile_cat: email!

done.